app/slave/cluster_slave.py

```diff
@@ -195,7 +199,7 @@ def _send_master_idle_notification(self):
         self._logger.info('Notifying master that this slave is ready for new builds.')
         self._notify_master_of_state_change(SlaveState.IDLE)
 
-    def _disconnect_from_master(self):
+    def disconnect_from_master(self):
```

Comment:
it's used in test_cluster_slave, I guess that's why I changed it.  I'll change it back